### ISSUE TITLE
I want to add a landing page example for a website in the projects folder

### ISSUE THREAD

**Issue by aryan04dbz** [you-dont-need/You-Dont-Need-JavaScript#494]
This is my project :point_down: 

https://aryanfylo.netlify.app/

[Screencast from 2024-10-05 11-54-44.webm](https://github.com/user-attachments/assets/7cae81c5-fbba-40d0-9c6e-bb5afe66a2d5)

**Comment by aayush105** [comment]
> This is my project 👇
> 
> https://aryanfylo.netlify.app/
> 
>  Screencast.from.2024-10-05.11-54-44.webm

Yes, you can add this project if its pure HTML and CSS under the website folder section with you folder name containing your html and css file.

**Comment by aryan04dbz** [comment]
Yes this is a pure HTML and CSS project

**Comment by aayush105** [comment]
> Yes this is a pure HTML and CSS project

Ok you can create a PR for this project and please add your project under the folder section where it belong to. Also before creating the PR please firstly update your branch and then pull the main branch and then create a PR to avoid the conflict. @aryan04dbz
